database/databaseinterface.go

```diff
@@ -5,20 +5,20 @@ import (
 )
 
 // Database contains all the functions that interact with the database.
-type Database interface {
-	// Creates a new file, stores it in the database, and returns the id and revision number for it.
-	CreateEmptyFile(fileName string, userName string) (int, int)
+type DatabaseInterface interface {
+	// Creates a new file, stores it in the database, and returns the id for it.
+	CreateEmptyFile(fileName string, userID string) (int, int, error)

```

Comment:
Could you revert the comment to mention the revision number? 😄 (unless we don't wanna return it?)

Reply:
**Fixed.**